bin/cron/cspschools.csv

```diff
@@ -0,0 +1 @@
510003201932,,APPOMATTOX REGIONAL GOV. SCH.,PETERSBURG,VA,23803,1,13
510009000015,470420,ALBEMARLE HIGH,CHARLOTTESVILLE,VA,22901,1,28
510084002661,470503,COSBY HIGH,MIDLOTHIAN,VA,23112,1,64
510126000059,470402,CHANTILLY HIGH,CHANTILLY,VA,20151,1,60
510180000731,471020,BETHEL HIGH,HAMPTON,VA,23666,1,37
510189000809,471070,HIGHLAND SPRINGS HIGH,HIGHLAND SPRINGS,VA,23075,1,22
510267001098,471585,GRANBY HIGH,NORFOLK,VA,23505,1,19
510267001119,471619,NORFOLK TECHNICAL CTR,NORFOLK,VA,23502,1,27
530006000013,480015,ADNA MIDDLE/HIGH SCHOOL,CHEHALIS,WA,98532,1,5
530033000043,481580,BAINBRIDGE HIGH SCHOOL,BAINBRIDGE ISLAND,WA,98110,1,12
530066000139,480103,BREMERTON HIGH SCHOOL,BREMERTON,WA,98337,1,17
530111000191,481483,CENTRAL VALLEY HIGH SCHOOL,VERADALE,WA,99037,1,41
530111000204,481297,UNIVERSITY HIGH SCHOOL,SPOKANE,WA,99216,1,16
530123000223,480185,CHENEY HIGH SCHOOL,CHENEY,WA,99004,1,5
530141000250,481370,CLOVER PARK HIGH SCHOOL,LAKEWOOD,WA,98499,1,11
530141000261,480588,LAKES HIGH SCHOOL,LAKEWOOD,WA,98498,1,14
530141003000,480589,HARRISON PREP SCHOOL,LAKEWOOD,WA,98498,1,10
530156000278,480480,COLUMBIA HIGH AND ELEMENTARY,HUNTERS,WA,99137,1,5
530228000312,481238,EAST VALLEY HIGH SCHOOL,SPOKANE VALLEY,WA,99216,1,11
530237000327,480335,EATONVILLE HIGH SCHOOL,EATONVILLE,WA,98328,1,13
530267000391,480383,CASCADE HIGH SCHOOL,EVERETT,WA,98203,1,24
530267000395,480385,EVERETT HIGH SCHOOL,EVERETT,WA,98201,1,56
530267001726,480711,HENRY M. JACKSON HIGH SCHOOL,MILL CREEK,WA,98012,1,35
530270003222,480139,UNION HIGH SCHOOL,CAMAS,WA,98607,1,26
530282003154,480401,TODD BEAMER HIGH SCHOOL,FEDERAL WAY,WA,98003,1,13
530288000465,481375,FIFE HIGH SCHOOL,TACOMA,WA,98424,1,26
530354000531,481090,HIGHLINE HIGH SCHOOL,BURIEN,WA,98148,1,6
530354003069,481159,ACADEMY OF CITIZENSHIP AND EMPOWERMENT,SEATAC,WA,98188,1,10
530354003180,481141,TECHNOLOGY ENGINEERING & COMMUNICATIONS,SEATTLE,WA,98146,1,13
530354003182,481139,ARTS & ACADEMICS ACADEMY,SEATTLE,WA,98146,1,22
530399001967,480540,KETTLE FALLS HIGH SCHOOL,KETTLE FALLS,WA,99141,1,6
530486001909,480674,MARYSVILLE MOUNTAIN VIEW HIGH SCHOOL,MARYSVILLE,WA,98270,1,6
530486003323,480675,INTL SCH OF COMMUNICATIONS,MARYSVILLE,WA,98270,1,13
530645000953,480855,ORTING HIGH SCHOOL,ORTING,WA,98360,1,22
530696001038,480940,PUYALLUP HIGH SCHOOL,PUYALLUP,WA,98371,1,37
530723001077,480985,LINDBERGH SENIOR HIGH SCHOOL,RENTON,WA,98058,1,18
530780003344,480495,SELKIRK HIGH SCHOOL,IONE,WA,99139,2,6
530792001317,481166,SHOREWOOD HIGH SCHOOL,SHORELINE,WA,98133,1,29
530825001385,481258,FERRIS HIGH SCHOOL,SPOKANE,WA,99223,1,26
530825001386,481260,ROGERS HIGH SCHOOL,SPOKANE,WA,99207,1,12
530825001388,481265,LEWIS & CLARK HIGH SCHOOL,SPOKANE,WA,99204,1,21
530825001407,481282,SHADLE PARK HIGH SCHOOL,SPOKANE,WA,99205,1,11
530825003378,481283,ON TRACK ACADEMY,SPOKANE,WA,99207,1,10
530927003415,481493,VANCOUVER ITECH PREPARATORY,VANCOUVER,WA,98686,1,20
530969001648,480710,WEST VALLEY HIGH SCHOOL,SPOKANE,WA,99212,1,12
```

Comment:
could we get newlines in here easily? :)

Reply:
Done. Word saved newlines as \r (MacOS standard). I changed it to \n (Linux standard) and that displays better in GitHub. (The CSV parser is happy with either.)